scala/tempo/src/main/scala/com/databrickslabs/tempo/TSSchema.scala

```diff
@@ -0,0 +1,97 @@
+package com.databrickslabs.tempo
+
+import org.apache.spark.sql.types.{NumericType, StructField, StructType}
+
+/**
+ * A Timeseries Schema
+ */
+trait TSSchema
```

Comment:
Can we switch to camelcase or rename? TSStruct and TSSchema are tricky to type out.

Reply:
I'll switch it to CamelCase for now - we can think about renaming if there are any good suggestions.

Reply:
this is done